src/FormattingSettingsComponents.ts

```diff
@@ -19,31 +19,97 @@ class NamedEntity {
+    name: string;
+
     slices?: Array<Slice>;
     container?: Container;
 
+    topLevelToggle?: powerbi.visuals.EnabledSlice;
+
+    disabled?: boolean;
+    /** group disabled reason */
+    disabledReason?: string;
+    /**
+             * If delaySaveSlices is true, then this group's slices' value changes won't be saved to the visual until a
+             * signal action is taken. E.g., for an Analytics Pane forecast, the forecast parameter values shouldn't be
+             * saved to the visual until the Apply button is clicked. Note that this applies to all slices in the group.
+             */
+    delaySaveSlices?: boolean;
+    /** Group can expand/collapse */
+    collapsible?: boolean;
+
+    /**
+     * Returns the formatting group.
+     * 
+     * To be used as an entry point for dynamic formatting groups.
+     */
+    getFormattingGroup?(objectName: string, localizationManager?: powerbi.extensibility.ILocalizationManager): visuals.FormattingGroup {
+        return {
+            displayName: (localizationManager && this.displayNameKey)
+                ? localizationManager.getDisplayName(this.displayNameKey) : this.displayName,
+            description: (localizationManager && this.descriptionKey)
+                ? localizationManager.getDisplayName(this.descriptionKey) : this.description,
+            slices: [],
+            topLevelToggle: this.topLevelToggle,
+            uid: this.name + "-group",
+            collapsible: this.collapsible,
+            delaySaveSlices: this.delaySaveSlices,
+            disabled: this.disabled,
+            disabledReason: this.disabledReason,
+        }
+    }
+}
+
+export class Card extends Group implements IFormattingSettingsCard {
+    constructor() {
+        super({ name: undefined });
+    }
+
     /** if true, this card should be populated into the analytics pane */
     analyticsPane?: boolean;
 
+     /**
+     * Returns the formatting card with single group.
+     * 
+     * To be used as an entry point for dynamic formatting cards.
+     */
     getFormattingCard?(objectName: string, group: visuals.FormattingGroup, localizationManager?: powerbi.extensibility.ILocalizationManager): visuals.FormattingCard {
         return {
             displayName: (localizationManager && this.displayNameKey)
                 ? localizationManager.getDisplayName(this.displayNameKey) : this.displayName,
             description: (localizationManager && this.descriptionKey)
                 ? localizationManager.getDisplayName(this.descriptionKey) : this.description,
             groups: [group],
+            topLevelToggle: this.topLevelToggle,
             uid: objectName,
-            analyticsPane: this.analyticsPane
+            analyticsPane: this.analyticsPane,
+            disabled: this.disabled,
+            disabledReason: this.disabledReason,
         }
     }
 }
 
+export type Cards = Card | CompositeCard;
```

Comment:
`Card` is already used by the current release https://github.com/microsoft/powerbi-visuals-utils-formattingmodel/blob/e14c2dac3fa0d6ba16107b22c388fbc304dbb1d7/src/FormattingSettingsComponents.ts#L25
`export type Card = SimpleCard | CompositeCard;` it will break backwards combability as the current `Card` class will be removed.

Reply:
@AleksSavelev what do you think about this?
Can we break backward compatability for Card class name ? the users that will move from old version to the new version of formatting model utils will have to replace their object of type card to simple card?

Reply:
@ShereinDabbah @AleksSavelev Any news on this? Should we break the backward compatibility for `Card`?

Reply:
@AleksSavelev Can you please answer this in here

Reply:
Let's revert these changes. It seems to be an optional change, so it is better not to break backward compatibility where it's possible

Reply:
Let's fix this comment and merge the PR

Reply:
Thanks for the update and to make it clear, we introduce the new type `Cards`?
```
export type Cards = SimpleCard | CompositeCard;
```

Reply:
Yes, let's add the `Cards` to keep the backward compatibility